pkg/storage/mongo/mongo.go

```diff
@@ -38,6 +35,9 @@ func (m *ModuleStore) Connect() error {
 		Background: true,
 		Sparse:     true,
 	}
-	c := m.s.DB(m.d).C(m.c)
-	return c.EnsureIndex(index)
+	c := sess.DB(db).C(coll)
+	if err := c.EnsureIndex(index); err != nil {
+		return nil, err
+	}
+	return ms, nil
```

Comment:
again I think it's better from a readability standpoint to explicitly pass the address of a value on return instead of declaring a variable of type pointer earlier on.

Reply:
agree - thanks!